tests/metrics/network/dockerfile_image/Dockerfile

```diff
@@ -0,0 +1,46 @@
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# XXX: named version
+FROM ubuntu
+
+# Ensure packages are current, then install:
+#
+# - a basic development environment
+# - runtime build dependencies and additional general tooling
+#
+RUN apt-get -y update && \
+    apt-get install -y build-essential && \
+    apt-get install -y curl
+
+# Install smem (Memory reporting tool)
+RUN apt-get install -y smem
```

Comment:
I'd be curious if others have input on this, though - it's more of a style thing.  @jodh-intel ?

Best practices link: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/